### PR TITLE
Frontend UI, checkout & generate routes

### PR DESCRIPTION
**Pull request by saikiransomanagoudar**
Added Pages:
1. Landing page
2. Sign-Up/Sign-In forms
3. Generate Flashcard page

### CODE REVIEW

**Review by andrewalson — APPROVED** [review]
Cleaned up a bit for you and this looks like a great start for us to go off of 👍

File: app/api/checkout_sessions/route.js
Comment (tanmay-sketch) [inline app/api/checkout_sessions/route.js:1]: looks good

File: app/api/generate/route.js
Comment (tanmay-sketch) [inline app/api/generate/route.js:1]: looks good

File: app/flashcards/page.js
Comment (tanmay-sketch) [inline app/flashcards/page.js:1]: looks good

File: app/generate/page.js
Comment (tanmay-sketch) [inline app/generate/page.js:1]: looks good

File: app/globals.css
Comment (tanmay-sketch) [inline app/globals.css:1]: most of the css is being added from the inline styling itself so this is fine

File: app/layout.js
Comment (tanmay-sketch) [inline app/layout.js:1]: layout.js for auth looks good

File: app/page.js
Comment (tanmay-sketch) [inline app/page.js:1]: looks good

File: app/pages/api/saveFlashcards.js
Comment (tanmay-sketch) [inline app/pages/api/saveFlashcards.js:1]: looks good

File: app/sign-up/[[...sign-up]]/page.js
Comment (tanmay-sketch) [inline app/sign-up/[[...sign-up]]/page.js:1]: sign up and sign in look good

File: app/sign-in/[[...sign-in]]/page.js
Comment (tanmay-sketch) [inline app/sign-in/[[...sign-in]]/page.js:1]: sign up and sign in look good

File: firebase.js
Comment (tanmay-sketch) [inline firebase.js:1]: API key and credentials are added to firebase.js, add to .env and remove it

File: utils/get-stripe.js
Comment (tanmay-sketch) [inline utils/get-stripe.js:1]: looks good

**Review by tanmay-sketch — CHANGES_REQUESTED** [review]
Everything looks good, remove the api key and credentials from the repo and try to remove the commits with the api key if possible

**Comment by andrewalson** [comment]
The Typography import in flashcards/page.js fixes the Vercel build errors.

File: firebase.js
Comment (saikiransomanagoudar) [reply firebase.js:1]: Removed the api key, and replaced with the variable name FIREBASE_API_KEY and stored it in .env.local file
image

**Review by tanmay-sketch — APPROVED** [review]
Changes look good

**Review by vamsi2792 — APPROVED** [review]
Everything looks good!

**Pull request merged**